### PR TITLE
add new GPU backend information to GPU data convert tutorial

### PR DESCRIPTION
**Pull request by slefrancois**

### CODE REVIEW

File: doc/tutorial/gpu_data_convert.txt [hunk 1]
```diff
@@ -4,19 +4,25 @@
...
 ======
 
-Currently, PyCUDA and Theano have different objects to store GPU
-data. The two implementations do not support the same set of features.
-Theano's implementation is called *CudaNdarray* and supports
-*strides*. It also only supports the *float32* dtype. PyCUDA's implementation
-is called *GPUArray* and doesn't support *strides*. However, it can deal with
-all NumPy and CUDA dtypes.
+Theano's new GPU backend (:ref:`theano.gpuarray<libdoc_gpuarray>`) is based on 
+PyCUDA's GPUArray objects. However, if you are still using the old backend,
```
Comment (nouiz) [inline doc/tutorial/gpu_data_convert.txt]: This is false. GpuArray aren't based on PyCUDA. But I think GPUArray have a compatible interface to PyCUDA (but not the reverse). @abergeron can you confirm?

Comment (slefrancois) [inline doc/tutorial/gpu_data_convert.txt]: For sure, GPUarray no longer uses CudaNdarray types. When I said "based on pycuda's objects", I meant that it's not the same, but shares most of the interface. I could just explicitely say compatible.

**Comment by slefrancois** [comment]
test this please

**Comment by slefrancois** [comment]
jenkins test this please

**Comment by slefrancois** [comment]
jenkins test this please

**Comment by slefrancois** [comment]
jenkins test this please

**Comment by slefrancois** [comment]
jenkins test this please

**Comment by slefrancois** [comment]
jenkins test this please

**Comment by slefrancois** [comment]
jenkins test this please

**Pull request closed without merging**